torch/utils/data/dataloader.py

```diff
@@ -123,7 +126,8 @@ def __init__(self, dataset, batch_size=1, shuffle=False, sampler=None,
                  batch_sampler=None, num_workers=0, collate_fn=None,
                  pin_memory=False, drop_last=False, timeout=0,
                  worker_init_fn=None, multiprocessing_context=None,
-                 generator=None):
+                 generator=None,
+                 persistent_workers=False):
```

Comment:
Also let's make this keyword only and check that users don't set `num_workers > 0` and `persistent_worker=True` :)